src/Nancy.Tests/Unit/NancyEngineFixture.cs

```diff
@@ -42,7 +43,7 @@ public NancyEngineFixture()
             this.negotiator = A.Fake<IResponseNegotiator>();
 
             A.CallTo(() => this.requestDispatcher.Dispatch(A<NancyContext>._, A<CancellationToken>._))
-                .Returns(CreateResponseTask(new Response()));
+                .Returns(Task.FromResult(new Response()));

```

Comment:
Got rid of `TaskHelpers.CompletedTask` and used `Task.FromResult` where I could.